### PR TITLE
Fix/soc 5569 Can't view user activities feed

### PR DESCRIPTION
**Pull request by boubaker**
Modify parameter ownerId of type Long and add Unit test

### CODE REVIEW

File: component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/ActivityDAOImpl.java [hunk 1]
```diff
@@ -49,14 +48,11 @@
 public class ActivityDAOImpl extends GenericDAOJPAImpl<ActivityEntity, Long> implements ActivityDAO {
   
   public List<ActivityEntity> getActivities(Identity owner, Identity viewer, long offset, long limit) throws ActivityStorageException {
```
Comment (thomasdelhomenie) [inline component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/ActivityDAOImpl.java]: viewer is not used anymore, it should be removed.

**Comment by boubaker** [comment]
> Works fine. But I added a comment, and I have a test failure :

hmm, byteman doesn't work on Windows. Anyway, you can add the annotation @MaxQueryNumber and retest please. thx

**Comment by thomasdelhomenie** [comment]
Annotation added.

**Comment by boubaker** [comment]
Merged

**Pull request closed without merging**